Sources/MUXSDKStatsInternal/AVPlayerItem+AccessLog.swift

```diff
@@ -0,0 +1,44 @@
+import AVFoundation
+
+@available(iOS 15, tvOS 15, *)
+extension AVPlayerItem {
+    nonisolated var indicatedBitratesInAccessLog: some BidirectionalCollection<Double> {
```

Comment:
Making sure I understand: 

If we have indicated bitrates in the access log already we return (that one's easy)

If we need to wait for some, how does this property behave? Does it wait for the first indicated bitrate and return that?

Reply:
... your new commit makes this much more clear.